app/experimenter/visualization/api/v3/views.py

```diff
@@ -36,6 +38,60 @@ class Statistic:
     COUNT = "count"
 
 
+class JetstreamDataPoint(BaseModel):
+    lower: int = None
+    upper: int = None
+    point: int = None
+    metric: str = None
+    branch: str = None
+    statistic: str = None
+    window_index: str = None
+    comparison: str = None
+
+
+class JetstreamData(BaseModel):
+    __root__: List[JetstreamDataPoint] = []
+
+    def __iter__(self):
+        return iter(self.__root__)
+
+    def append(self, item):
+        self.__root__.append(item)
+
+    def extend(self, item):
+        self.__root__.extend(item)
+
+    def dict(self, exclude_none):
+        return [item.dict(exclude_none=exclude_none) for item in self.__root__]
```

Comment:
Huh interesting.... you can't just extend `List` as the base class for this? Or does it need to extend BaseModel?  Maybe you can mix them together? `(List, BaseModel)`?

Reply:
It won't let me do that, sadly. `TypeError: multiple bases have instance lay-out conflict`. Sounds like the two base classes do not cooperate